src/app/core/models/rules-interfaces.model.ts

```diff
@@ -0,0 +1,29 @@
+export class Rule{
+  name: string = '';
+  id?: number;
+  created_at?: Date;
+  tardiness_rules_attributes?:TradinessRule[] = [];
+  deleted_tardiness_rules?: TradinessRule[] = [];
+  leave_type_id?: number;
+  is_half_day?: boolean;
+  }
+
+ export class TradinessRule{
+    constructor(
+      public start_time: string = '' ,
+      public end_time: string = '',
+      public leave_type_id: number = 1,
+      public is_half_day: any = false,
+      public lop: number = 1,
```

Comment:
What does "lop" stand for?

Reply:
lop is 1 if the  is_half_day boolean is false and 0.5 if it's true